drivers/SmartThings/matter-sensor/profiles/air-quality-ikea.yml

```diff
@@ -0,0 +1,32 @@
+name: air-quality-ikea
```

Comment:
The name should indicate which capabilities it includes vs be device specific.

Reply:
Updated to `air-quality-sensor-tvoc-level` in 2f6d6be9343a806c29c8f9bfaa5122623aa0d116.